Extension/package.json

```diff
@@ -614,6 +623,16 @@
         "command": "C_Cpp.RescanWorkspace",
         "title": "%c_cpp.command.rescanWorkspace.title%",
         "category": "C/C++"
+      },
+      {
+        "command": "C_Cpp.VcpkgClipboardInstallSuggested",
+        "title": "%c_cpp.command.vcpkgClipboardInstallSuggested.title%",
+        "category": "C/C++"
+      },
+      {
+        "command": "C_Cpp.VcpkgOnlineHelpSuggested",
```

Comment:
I'm unclear as to whether this is meant to be an internal command or not.

Reply:
I've added it to the other necessary places